#request

---
#response
CapacityMsg capacity_total
CapacityMsg capacity_free
